Improve documention how to obtain the peering id (#2472)
mongodb/terraform-provider-mongodbatlas: docs/resources/network_peering.md

$ terraform import mongodbatlas_network_peering.my_peering 1112222b3bf99403840e8934-5cbf563d87d9d67253be590a-AWS
```

> To obtain `project_id` and `peering_id` can be obtained using atlas cli. Attention: `atlas networking peering list` needs the `--provider` parameter for AZURE and GCP as it returns only `AWS` peerings by default. 

Reviewer: ```suggestion
To obtain your `project_id` and `peering_id` with the atlas cli. Attention: The `atlas networking peering list` command returns `AWS` peerings by default. You must include the the `--provider` parameter for AZURE and GCP peerings. 
```

Reviewer 2: Applied the suggestion here https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2488

> To obtain `project_id` and `peering_id` can be obtained using atlas cli. Attention: `atlas networking peering list` needs the `--provider` parameter for AZURE and GCP as it returns only `AWS` peerings by default. 

Reviewer 3: replace "atlas cli" with "MongoDB Atlas CLI[https://www.mongodb.com/docs/atlas/cli/current/command/atlas-networking-peering-list/#std-label-atlas-networking-peering-list]"

Reviewer 3: also would leverage language we already use in CLI Docs for consistency for description of the --provider parameter: Name of your cloud service provider. Valid values are AWS, AZURE, or GCP. This value defaults to "AWS".

Reviewer 2: Changed it in https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2488
